nucypher/keystore/keystore.py

```diff
@@ -145,9 +158,24 @@ def del_policy_arrangement(self, arrangement_id: bytes, session=None):
         session.query(PolicyArrangement).filter_by(id=arrangement_id).delete()
         session.commit()
 
+    def del_expired_policy_arrangements(self, session=None, now=None) -> int:
+        """
+        Deletes all expired PolicyArrangements from the Keystore.
+        """
+        session = session or self._session_on_init_thread
+        now = now or datetime.now()
+        result = session.query(PolicyArrangement).filter(PolicyArrangement.expiration < now)
```

Comment:
```suggestion
        result = session.query(PolicyArrangement).filter(PolicyArrangement.expiration <= now)
```

Gotta make this a `<=` comparison.

Reply:
:heavy_check_mark: